src/covid19sim/server_utils.py

```diff
@@ -407,55 +407,55 @@ def proc_human(params, inference_engine=None):
         if USE_MESSAGE_BATCHING:
             encounter_messages = covid19sim.frozen.utils.convert_messages_to_batched_new_format(
-                human["messages"], human["exposure_message"])
+                human.messages, human.exposure_message)
             update_messages = covid19sim.frozen.utils.convert_messages_to_batched_new_format(
-                human["update_messages"])
+                human.update_messages)
             earliest_new_encounter_message = encounter_messages[0][0] if len(encounter_messages) else None
         else:
             encounter_messages = \
-                [covid19sim.frozen.utils.convert_message_to_new_format(m) for m in human["messages"]]
+                [covid19sim.frozen.utils.convert_message_to_new_format(m) for m in human.messages]
             exposure_message = \
-                covid19sim.frozen.utils.convert_message_to_new_format(human["exposure_message"]) \
-                if human["exposure_message"] else None
+                covid19sim.frozen.utils.convert_message_to_new_format(human.exposure_message) \
+                if human.exposure_message else None
             # since the original messages do not carry the 'exposition' flag, we have to set it manually:
             # hopefully this will only match the proper messages (based on _real_sender_id comparisons)...
             if exposure_message is not None:
                 for m in encounter_messages:
                     m._exposition_event = m == exposure_message
             update_messages = \
-                [covid19sim.frozen.utils.convert_message_to_new_format(m) for m in human["update_messages"]]
+                [covid19sim.frozen.utils.convert_message_to_new_format(m) for m in human.update_messages]
             earliest_new_encounter_message = encounter_messages[0] if len(encounter_messages) else None
         if earliest_new_encounter_message is not None:
             # quick verification: even with a 1-day buffer for timeslot craziness, we should not be
             # getting old encounters here; the simulator should not keep & transfer those every call
             assert REBUILD_CLUSTERS_FROM_SCRATCH or \
                    earliest_new_encounter_message.encounter_time + 1 >= params["current_day"]
-        human["clusters"].add_messages(
+        human.clusters.add_messages(
             messages=[*encounter_messages, *update_messages],
             current_timestamp=params["current_day"],
         )
 
     # Format for supervised learning / transformer inference
-    todays_date = params["start"] + datetime.timedelta(days=params["current_day"])
-    is_exposed, exposure_day = covid19sim.frozen.helper.exposure_array(human["infection_timestamp"], todays_date)
-    is_recovered, recovery_day = covid19sim.frozen.helper.recovered_array(human["recovered_timestamp"], todays_date)
+    todays_date = params["start"] + datetime.timedelta(days=params["current_day"], hours=params["time_slot"])
```

Comment:
@pg2455 , @vict0rsch , this fixes a small bug where today's data were excluded. `(todays_date - some_datetime).days` could lead to a negative value which caused excluding the data from the current day's output.